src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/DRConfig/SetAzureEventHubDRConfigurationFailOver.cs

```diff
@@ -0,0 +1,57 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.EventHub.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.EventHub.Commands.GeoDR
+{
+    /// <summary>
+    /// 'New-AzureRmEventHubDRConfigurationsFailOver' Cmdlet envokes GEO DR failover and reconfigure the alias to point to the secondary namespace
```

Comment:
@v-Ajnava same typo: `New` instead of `Set`

Reply:
Resolved